### PR TITLE
Feature #256 adding the component filters.

### PR DESCRIPTION
**Pull request by kenjishiromajp**

### CODE REVIEW

**Comment by jaortiz** [comment]
@kenjishiromajp are we able to add a demo to show the filter working with content?

**Comment by kenjishiromajp** [comment]
> @kenjishiromajp are we able to add a demo to show the filter working with content?

yeah sure, I was literally copying the examples from the previous repo.

File: packages/ui/src/components/filter/components/buttons/buttons.component.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+import { styles } from './buttons.styles.js';
+import { type ButtonsProps } from './buttons.types.js';
+
+export function Buttons({
+  filterButtons,
+  onClick,
+  selectedButton,
+  resultsFound,
+  tag: Tag = 'nav',
```
Comment (samithaf) [inline packages/ui/src/components/filter/components/buttons/buttons.component.tsx:14]: Not sure `nav` needs to be the default tag

Comment (samithaf) [reply packages/ui/src/components/filter/components/buttons/buttons.component.tsx:14]: Just looking at Google implementation here, https://material-web.dev/components/chip/stories/

**Review by samithaf — APPROVED** [review]

**Pull request merged**